src/sunrise/core.cljs

```diff
@@ -1,85 +1,13 @@
 (ns sunrise.core
-  (:require [reagent.core :as r]))
+  (:require [reagent.core :as reagent]
+            [sunrise.component :as component]))
```

Comment:
We’ll probably come up with semantically distinct names for these components as we get more. Maybe the top level svg is scene and the mountains live in a mountain component. Probably can happen in a future pr.